Dockerfile-2.7

```diff
@@ -0,0 +1,47 @@
+FROM python:2.7-alpine3.6 
+
+LABEL maintainer="Arista Ansible Team <ansible@arista.com>"
+LABEL com.example.version="0.5"
+LABEL vendor1="Arista"
+LABEL com.example.release-date="2019-10-29"
+LABEL com.example.version.is-production="False"
+
+ENV PS1='py2.7-ansible-cvp:\u% '
+
+WORKDIR /tmp
+WORKDIR /tmp
+
+RUN apk add --update --no-cache ca-certificates \
+                                openssh-client \
+                                build-base \
+                                gcc \
+                                g++ \
+                                make \
+                                python-dev \
+                                py-pip \
+                                libffi-dev \
+                                sshpass \
+                                libssl1.0 \
+                                openssl-dev
+
+COPY requirements.txt .
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt
+
+RUN apk del -r --purge gcc make g++ &&\
+    rm -rf /source/* &&\
+    rm -rf /var/cache/apk/* &&\
+    rm -rf /tmp/*
```

Comment:
it's better to combine these in a single layer, i.e. add `&& \` and make then a part of the same RUN command. this should reduce the image size

Reply:
Indeed, new build give an image of `330MB` without build option. 
Going to be updated accordingly